include/fmt/format.h

```diff
@@ -3317,6 +3317,26 @@ FMT_CONSTEXPR20 void format_hexfloat(Float value, int precision,
   format_hexfloat(static_cast<double>(value), precision, specs, buf);
 }
 
+template <typename Float> FMT_CONSTEXPR auto iceil(Float value) -> int {
+  auto min = (std::numeric_limits<int>::min)();
+  auto max = (std::numeric_limits<int>::max)();
+  ignore_unused(min, max);
+  FMT_ASSERT(value >= min && value <= max, "value not in int range");
+  if (is_constant_evaluated()) {
```

Comment:
On a second thought, I don't think we need this check at all because `iceil` is not called at runtime except in tests.

Reply:
Done